src/volttron/client/commands/install_agents.py

```diff
@@ -453,14 +418,19 @@ def send_zmq():
             channel.close(linger=0)
             del channel
 
+    task = None
     if server.core.messagebus == "rmq":
-        _log.debug(f"calling install_agent on {peer} sending to topic {rmq_send_topic}")
-        task = gevent.spawn(send_rmq)
-        # TODO: send config
+        if wheel_install:
+            _log.debug(f"calling install_agent on {peer} sending to topic {rmq_send_topic}")
+            task = gevent.spawn(send_rmq)
+            # TODO: send config
```

Comment:
Is this TODO done due to this PR?

Reply:
this is another feature to be done. vctl install does not have all features of monolithic code yet